pypeit/spectrographs/gemini_gmos.py

```diff
@@ -333,7 +333,7 @@ def get_rawimage(self, raw_file, det):
             array[row1-nodpix:row2-nodpix,:] = array[row1:row2,:]
 
```

Comment:
Hi @deborape, thanks for catching this. I fear this bug may arise in other spectrograph files that simply copied the one with the bug, which is usually what is done when we add new spectrographs. Can you via grep or brute force, just make sure this bug is caught in other spectrographs. I worry we may not have a test for it to fail, but that the bug is still there. Maybe you did this already which is how you find these other cases, in which case ignore this request.

Reply:
Yes, I believe I checked all the spectrographs where this bug may have propagated.